cmd/fluxctl/list_images_cmd.go

```diff
@@ -31,7 +31,7 @@ func (opts *controllerShowOpts) Command() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "list-images",
 		Short:   "Show the deployed and available images for a controller.",
-		Example: makeExample("fluxctl list-images --namespace default --controller=deployment/foo"),
+		Example: makeExample("fluxctl list-images --namespace default --controller=default:deployment/foo"),
```

Comment:
The idea in general is that you provide `--namespace` to give the namespace, and `--controller` to name the resource within that namespace, to be a bit more like kubectl.

The `namespace:kind/name` syntax is mainly because we refer to things elsewhere in that way -- perhaps we shouldn't -- and it's convenient. In any case, examples ought not use both the namespace argument and the namespace-specifying syntax.

Reply:
(How this works may need rethinking; nonetheless, it's how it works :)

Reply:
Thanks, I overlooked these namespace args, was replacing blindly.